### PR TITLE
War-2066(Katana: Changes in Katana UI for setup and cleanup support.)

### PR DESCRIPTION
**Pull request by apathapa-warrior**
Added logs in the jira ticket

### CODE REVIEW

File: katana/config/newtestcase.xml [hunk 1]
```diff
@@ -7,11 +7,13 @@
 		<Time></Time>
```
Comment (santhosh-warrior) [inline katana/config/newtestcase.xml]: Is the file required ?

File: katana/static/app/controllers/testcaseCapCtrl.js [hunk 2]
```diff
@@ -1299,7 +1447,7 @@ $scope.showRules = function(execType){
                 else{
                     if($scope.status.nodatafile == '0'){
                     swal({
-                        title: "Kindly provide the correct Relative path for Input data File, if auto-population of system & Subsystem name is needed.",
+                        title: "Kindly provide the correct Relative path for Input data File and Jocket File, if auto-population of system & Subsystem name is needed.",
```
Comment (santhosh-warrior) [inline katana/static/app/controllers/testcaseCapCtrl.js]: Use Wrapper file

File: warrior/Warriorspace/wrapper_files/tw_sample.xml [hunk 3]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" ?>
```
Comment (santhosh-warrior) [inline warrior/Warriorspace/wrapper_files/tw_sample.xml]: The same is file is added from 2041 commit also.

File: katana/config/newtestwrapperfile.xml [hunk 4]
```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" ?>
```
Comment (santhosh-warrior) [inline katana/config/newtestwrapperfile.xml]: Is the required file ?  File is already present under Warriorspace.

File: katana/config/newtestcase.xml (hunk 1 above)
Comment (apathapa-warrior) [reply katana/config/newtestcase.xml]: Hi @santhosh-warrior 
This file is already presented. I  added the tag for wrapper file.

File: katana/config/newtestwrapperfile.xml (hunk 4 above)
Comment (apathapa-warrior) [reply katana/config/newtestwrapperfile.xml]: Hi @santhosh-warrior ,

we are using this file as a reference(tag names, setup and cleanup blocks) while generating wrapper file. We cannot use wrapper file available in the warriorspace/wrapperfiles as the content available in the tw_sample will be shown as default in UI.

File: katana/static/app/controllers/testcaseCapCtrl.js (hunk 2 above)
Comment (apathapa-warrior) [reply katana/static/app/controllers/testcaseCapCtrl.js]: -Made necessary changes according to your suggestions and committed.

File: warrior/Warriorspace/wrapper_files/tw_sample.xml (hunk 3 above)
Comment (apathapa-warrior) [reply warrior/Warriorspace/wrapper_files/tw_sample.xml]: Removed from 2041 branch.

**Pull request merged**